fuel_test/fuel_test.sh

```diff
@@ -2,6 +2,17 @@
 
 set -eux
 
+# If XCP is 1.9.0 (XS6.54) and branch is master, no need test
```

Comment:
I think this just requires a decision. If we just skip XS6.5 testing on master branch since this moment (assume master branch is for MOS10). Then I can change zuul to filter jobs basing on the branches.